src/test/parser/metadata.unit.test.ts

```diff
@@ -42,7 +42,7 @@ suite('Unit Test: Metadata Parser', () => {
         const testAndExpectCode = async (filename: string, code: string) => {
             await assert.rejects(
                 () => parseMetadata(path.join(staticPath, filename)),
-                (err) => {
+                (err: MetadataParseError) => {
```

Comment:
Shouldn't you use the MetadataParseError in the next test case too?

Reply:
This change was just to silence a TS type-error because it couldn't deduce the error type of parseMetadata.
Might as well use the type on the next case as well, fixed.